cmd/microcloud/ask.go

```diff
@@ -715,8 +709,9 @@ func (c *initConfig) askRemotePool(sh *service.Handler) error {
 				targetDisks[target] = append(targetDisks[target], path)
 			}
 
-			if !useJoinConfigRemote && len(targetDisks) < 3 {
-				return fmt.Errorf("Unable to add remote storage pool: At least 3 peers must have allocated disks")
+			insufficientDisks := !useJoinConfigRemote && len(targetDisks) < recommendedDiskSpread
+			if c.autoSetup && insufficientDisks {
```

Comment:
Is this producing a warning (like mentioned in the commit message) or does it re-ask the question if I select fewer 3 disks? Same for the check in interactive mode below.

Reply:
`askRetry` skips the question and exits if `c.autoSetup` is true.

With `--auto`, if your systems don't have enough resources to set up at least 3 disks across 3 systems, then `remote` storage pool setup will be skipped entirely with that message.

Reply:
Mh but in interactive mode when I select three disks on fewer than three systems I'll get the `Disk configuration does not meet recommendations for fault tolerance` message as it returns the error to `askRetry()`. 

To be sure I just checked out your branch and tried myself. Wasn't aware of the fact that you have to say `no` after getting the message and it will then continue on with your selection of fewer than three systems.

Would it make sense to add another sentence saying you can continue on but this wouldn't protect you from potential data loss?
At least I got confused by it :)

Reply:
Sure, I'll add an additional line. It'll be weird since the error will have a `\n` in it but that should be OK.

Reply:
The error is now printed like this:
```
Disk configuration does not meet recommendations for fault tolerance. At least 3 systems must supply disks.
Continuing with this configuration will leave MicroCloud susceptible to data loss
Change disk selection? (yes/no) [default=yes]:  
```

Reply:
Thanks, looks good to me!